src/info.cr

```diff
@@ -17,6 +17,11 @@ class Shards::Info
 
   def save
     Dir.mkdir_p(@install_path)
+
+    unless File.exists?(info_path)
+      Dir[File.join @install_path, "*.sha1"].each { |p| File.delete(p) }
```

Comment:
This is incorrect: Glob patterns must always use `/` as separator, not `\` even on Windows.